docs/serverless/02-01-serverless.md

```diff
@@ -6,6 +6,8 @@ Serverless relies on [Knative Serving](https://knative.dev/docs/serving/) for de
 
 ![Serverless architecture](./assets/serverless-architecture.svg)
 
+> **CAUTION:** Serverless imposes some requirements on the setup of Namespaces. If you create a new Namespace, do not disable sidecar injection in it as Serverless requires Istio for other resources to communicate with functions correctly. Also, if you apply custom [LimitRanges](https://kyma-project.io/docs/#details-resource-quotas) for a new Namespace, they must be higher than the default values.
```

Comment:
`they must be higher than the default values.` -> `they must be higher or equal than the default values.` and link to default values

Reply:
> ... they must be higher than the default values

default values for what -> please specify that we are talking about build  step,resources for build jobs

Reply:
yup, corrected